packages/linter/README.md

```diff
@@ -0,0 +1,117 @@
+# AMP Linter
+
+[![npm
+version](https://badge.fury.io/js/amp-toolbox-linter.svg)](https://badge.fury.io/js/amp-toolbox-linter)
+
+## Overview
+
+A [linter](<https://en.wikipedia.org/wiki/Lint_(software)>) for AMP documents:
+reports errors and suspicious constructions such as images missing or
+incorrectly sized, missing CORS headers, or invalid metadata.
+
+## Status
+
+This code is alpha quality.
+
+## Usage
+
+Command-line (local build):
+
+```sh
+$ npm install
+$ npm run build # generates src/cli.js from src/cli.ts
+$ node src/cli.js https://www.ampproject.org/
+```
+
+Command-line (from npm):
+
+```sh
+$ npx amp-toolbox-linter https://www.ampproject.org/
```

Comment:
s/ampproject.org/amp.dev

Reply:
Fixed in b8b1612be430efd525d598cbfeb7525eb090a8c2.